app/views/state_file/questions/esign_declaration/edit.html.erb

```diff
@@ -8,6 +8,8 @@
     <%= t(".md.body_html") %>
   <% elsif current_state_code == "ny" %>
     <%= t(".ny.body_html") %>
+  <% elsif current_state_code == "nc" %>
+    <%= t(".nc.body_html") %>
```

Comment:
not ideal, i think we can make this dynamic once all states have their esign language in here